### DISCUSSION TITLE
understanding RTH parameter

### DISCUSSION THREAD

**Discussion by samsab1** [iNavFlight/inav-configurator#2267] [General]
too many parameters kill the parameters, it is well known .....
joking aside, RTH under INAV remains a mystery with all its parameters. Even if overall the use of RTH is satisfactory, I can not understand the subtleties and choose the right parameter according to the desired use (return following the starting tracking, landing or not, Configuration of the failsave with the RTH) etc ....
where could I find a translation by image of each possible scenario?
thanks for your help

**Comment by MrD-RC** [comment]
RTH is fully documented in the Wiki

https://github.com/iNavFlight/inav/wiki/Navigation-Mode%3A-Return-to-Home
